### PR TITLE
Added link users by email rule

### PR DESCRIPTION
**Pull request by ntotten**

### CODE REVIEW

**Comment by woloski** [comment]
also we should check if `email_verified` is true

**Comment by ntotten** [comment]
@woloski done, added check for verified emails

File: rules/link-users-by-email.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+This rule will link any accounts that have the same email address. You will need to create an API token to use this rule and set a configuration value for `AUTH0_API_TOKEN`. The token you create will need the `read:users` and `update:users` scopes.
+
+> Note: When linking accounts, only the metadata of the target user is saved. If you want to merge the metadata of the two accounts you must do that manually. See the document on [Linking Accounts](https://auth0.com/docs/link-accounts) for more details.
+
+```js
+function (user, context, callback) {
+  // Check if email is verified, we shouldn't automatically
+  // merge accounts if this is not the case.
+  if (!user.email_verified) {
+    callback(null, user, context);
```
Comment (woloski) [inline rules/link-users-by-email.md]: return missing

**Comment by ntotten** [comment]
@woloski fixed

**Pull request merged**